submissions/m-ruslan/a-tiny-JS-world/index.js

```diff
@@ -0,0 +1,66 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/m-ruslan/a-tiny-JS-world
+   Web app: https://m-ruslan.github.io/a-tiny-JS-world/
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+let dog = {
+  type: "animal",
+  legs: 4,
+  name: "Ghost",
+  saying: "bark",
+  friendTo: ["Monica", "Tom", "Everybody"],
+};
+let cat = {
+  type: "animal",
+  legs: 4,
+  name: "Tom",
+  saying: "meow",
+  friendTo: ["Monica", "Chandler", "Everyone who feeds"],
+};
+let woman = {
+  type: "human",
+  legs: 2,
+  hands: 2,
+  name: "Monica",
+  saying: "Hello",
+  friendTo: ["Chandler", "Ghost"],
+};
+let man = {
+  type: "human",
+  legs: 2,
+  hands: 2,
+  name: "Chandler",
+  saying: "Hi",
+  friendTo: ["Monica", "Tom"],
+};
+
+// ======== OUTPUT ========
+/* Use print(message) for output.
+   Default tag for message is <pre>. Use print(message,'div') to change containing element tag.
+
+   Message can contain HTML markup. You may also tweak index.html and/or styles.css.
+   However, please, REFRAIN from improving visuals at least until your code is reviewed
+   so code reviewers might focus on a single file that is index.js.
+   */
+
+let makeMessage = (obj) =>
+  Object.keys(obj)
```

Comment:
You had explicit list of properties. With `Object.keys` it became implicit, which is not really good for some reasons:
- we want to know what exactly properties gonna be printed
- we may want to control what exactly props to use and what to skip
- order of keys is not guaranteed

Please fix.

Reply:
Oleksiy, thank you
I've already sent to review task ["post-OOP"](https://github.com/kottans/frontend-2021-homeworks/pull/220) and changed there `Object.keys` to `for...in` loop, but I'm not sure that this is the right answer here (also added `for...in` loop to this PR).

Could you please check one more time?)